### PR TITLE
docs: add sso part in function readme

### PR DESCRIPTION
**Pull request by KennethBWSong**
Add description to SSO in function readme

### CODE REVIEW

File: templates/function-base/js/default/README.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ Azure Functions are a great way to add server-side behaviors to any Teams applic
 - An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
 - [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) or [TeamsFx CLI](https://aka.ms/teamsfx-cli)
 
+## Single Sign On
+
+Microsoft Teams has provided a mechanism to minimize the number of times users need to enter their sign in credentials and this is called single sign on. Teams Framework (TeamsFx) added support on top of this mechanism to help developers build single sign feature easily. To support authentication in Azure Function, Single Sign On will be added to your project when you add Azure Function. See [Wiki](https://aka.ms/teamsfx-add-sso) to learn more about Single Sign On feature.
+
```
Comment (MuyangAmigo) [inline templates/function-base/js/default/README.md:14]: Same as here: https://github.com/OfficeDev/TeamsFx-Samples/pull/369#discussion_r870431121

File: templates/function-base/ts/default/README.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ Azure Functions are a great way to add server-side behaviors to any Teams applic
 - An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
 - [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) or [TeamsFx CLI](https://aka.ms/teamsfx-cli)
 
+## Single Sign On
+
+Microsoft Teams has provided a mechanism to minimize the number of times users need to enter their sign in credentials and this is called single sign on. Teams Framework (TeamsFx) added support on top of this mechanism to help developers build single sign feature easily. To support authentication in Azure Function, Single Sign On will be added to your project when you add Azure Function. See [Wiki](https://aka.ms/teamsfx-add-sso) to learn more about Single Sign On feature.
```
Comment (MuyangAmigo) [inline templates/function-base/ts/default/README.md:13]: Same as here: https://github.com/OfficeDev/TeamsFx-Samples/pull/369#discussion_r870431121

File: templates/function-base/js/default/README.md (hunk 1 above)
Comment (MuyangAmigo) [reply templates/function-base/js/default/README.md:14]: Also I'm not quite sure why we want to add this section in Function README?

File: templates/function-base/js/default/README.md [hunk 3]
```diff
@@ -8,6 +8,10 @@ Azure Functions are a great way to add server-side behaviors to any Teams applic
 - An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
 - [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) or [TeamsFx CLI](https://aka.ms/teamsfx-cli)
 
+## Single Sign On
+
```
Comment (MuyangAmigo) [inline templates/function-base/js/default/README.md:12]: We can say something like: `After adding function to your project, SSO will also be enabled. You can follow this [document](https://aka.ms/teamsfx-add-sso) to update your tab or bot code to include SSO.`

**Review by MuyangAmigo — APPROVED** [review]

**Review by hund030 — APPROVED** [review]

**Pull request merged**